core-tests/shared/src/test/scala/zio/ZIOSpec.scala

```diff
@@ -11,7 +10,7 @@ import zio.internal.Platform
 import zio.random.Random
 import zio.test.Assertion._
 import zio.test.TestAspect.{ flaky, forked, ignore, jvm, jvmOnly, nonFlaky, scala2Only }
-import zio.test._
+import zio.test.{ suite, _ }
```

Comment:
No need to import one member and then do wildcard import.

Reply:
👍